daemon/Cargo.toml

```diff
@@ -12,9 +12,8 @@ miningpool_observer_shared = { path="../shared" }
 log = "0.4"
 simple_logger = "1.9.0"
 
-bitcoin = "0.30"
-bitcoin-pool-identification = "0.2.5"
-rawtx-rs = { version = "0.1.6", features = [ "counterparty" ]}
+bitcoin-pool-identification = { git = "https://github.com/0xB10C/rust-bitcoin-pool-identification.git", branch = "2023-12-online-update-with-fallback"}
```

Comment:
Once https://github.com/0xB10C/rust-bitcoin-pool-identification/pull/6 is merged, update this

Reply:
done